packages/amplify-category-api/src/provider-utils/awscloudformation/service-walkthroughs/appSync-rds-walkthrough.js

```diff
@@ -167,26 +167,38 @@ async function selectDatabase(context, inputs, clusterArn, secretArn, AWS) {
   // Database Name Question
   const DataApi = new AWS.RDSDataService();
   const params = new DataApiParams();
+  const databaseList = [];
   params.secretArn = secretArn;
   params.resourceArn = clusterArn;
   params.sql = 'SHOW databases';
 
   spinner.start('Fetching Aurora Serverless cluster...');
-  const dataApiResult = await DataApi.executeStatement(params).promise();
 
-  // eslint-disable-next-line prefer-destructuring
-  const records = dataApiResult.records;
-  const databaseList = [];
+  try {
+    const dataApiResult = await DataApi.executeStatement(params).promise();
 
-  for (let i = 0; i < records.length; i += 1) {
-    const recordValue = records[i][0].stringValue;
-    // ignore the three meta tables that the cluster creates
-    if (!['information_schema', 'performance_schema', 'mysql'].includes(recordValue)) {
-      databaseList.push(recordValue);
+    // eslint-disable-next-line prefer-destructuring
+    const records = dataApiResult.records;
+
+    for (let i = 0; i < records.length; i += 1) {
```

Comment:
nit: I know this is how it was before the change, but can we simplify it?

```js
  for (const record of records) {
    const recordValue = record[0].stringValue;
```

Reply:
Updated!